bin/gopro-dashboard.py

```diff
@@ -65,6 +66,8 @@ def create_desired_layout(dimensions, layout, layout_xml, include, exclude, rend
     else:
         raise ValueError(f"Unsupported layout {args.layout}")
 
+def use_just_gpx(args):
+    return args.gpx and args.input == "-"
```

Comment:
probably we should add one more check here:
` and args.generate == "overlay"`.
What do you think?